### ISSUE TITLE
Is it possible to bind two keys to the same button?

### ISSUE THREAD

**Issue by hooliowobbits** [dajusc/Linux-Mouse-Keybinds#1]
Hey mate, thanks heaps for the work you have done on this.  I am currently using https://github.com/krayon/ratslap to configure my g300s.  It works ok, but I'm unable to use it to bind macros/combos (ie multiple keypresses to a single button).  Is that something that could work with Linux-Mouse-Keybinds?

**Some other things i found using your code**
I had to read up on evdev doco to determine i needed to be in the input group (ubuntu 18.04).  I also had to use their doco to work out what device name to use:
```
In [1]: import evdev                                                                                    
In [2]: devices = [evdev.InputDevice(path) for path in evdev.list_devices()]                            
In [3]: for device in devices: 
   ...:     print(device.path, device.name, device.phys) 
```
But that list reported multiple instances of my mouse, which was weird:
```
/dev/input/event3 Logitech G300s Optical Gaming Mouse usb-0000:00:12.2-3.2/input1
/dev/input/event2 Logitech G300s Optical Gaming Mouse usb-0000:00:12.2-3.2/input0
```
next, if i made a mistake in the button name while I got a useful list of buttons:
```
ERROR: Invalid button name. Must be one of: dict_keys(['BTN_LEFT', 'BTN_MOUSE', 'BTN_RIGHT', 'BTN_MIDDLE', 'BTN_SIDE', 'BTN_EXTRA', 'BTN_FORWARD', 'BTN_BACK', 'BTN_TASK', 'REL_X+', 'REL_X-', 'REL_Y+', 'REL_Y-', 'REL_WHEEL+', 'REL_WHEEL-'])
```
i wasn't able to work out which name mapped to which actual button.  And finally, if i set things up as follows and just spammed all the buttons, it didn't actually work :(
```
    lmkb = linuxmousekeybinds("Logitech G300s Optical Gaming Mouse")
    lmkb.bind_key_to_button(None, "REL_X+", "w")  #
```
Thoughts?

**Comment by dajusc** [comment]
Hey @hooliowobbits
its nice to hear that someone beside me uses LMK. :)

> [...] bind macros/combos (ie multiple keypresses to a single button). Is that something that could work with Linux-Mouse-Keybinds?

Right now it's not possible, but it shouldn't be much of a problem to implement this. I'll note this as a feature request.

> I had to read up on evdev doco to determine i needed to be in the input group (ubuntu 18.04).

Good hint, I'll add a remark in the documentation.

> I also had to use their doco to work out what device name to use

If you create the linuxmousekeybinds instance with a non-existing device, it will give you an error and list all available devices.

> But that list reported multiple instances of my mouse, which was weird:

Same for my Logitech G500s. Seem to be multiple devices (pointer + keyboard?) on that logical level.

> next, if i made a mistake in the button name while I got a useful list of buttons: [...]
> i wasn't able to work out which name mapped to which actual button. 

Its a bit of try&error for each mouse, because the names of the btn-events do not allways match the geometrical position of the actual hardware buttons. Take for example the Logitech G502 Lightspeed:

```
BTN_SIDE --> thumb button backward
BTN_EXTRA --> thumb button forward
BTN_FORWARD --> thumb button low
BTN_BACK --> Unassigned
```
Just create a test configuration assigning "1", "2", "3" etc to all available BTN events and test the result in a text-editor.

**Comment by dajusc** [comment]
hint about input group was added with 2ee1c1438edd1be16ebf20d212d06e890ad96fbc

**Comment by dajusc** [comment]
Macros are implemented with 336eab640266299228d69d16141b2327e02f7462
Issue with privileges for evdev described in README in 8d51441517414269ef1d71fe980917f16ec80eeb
